ClassHomework/src/APCSWork/Triangle.java

```diff
@@ -0,0 +1,68 @@
+package APCSWork;
+
+import java.util.ArrayList;
+
+public class Triangle extends Polygon{
+	public Triangle() {
+		super(3, new ArrayList<Point>());
```

Comment:
apparently in the starter code that I wrote, there is something like this: 

```
public Polygon(int numOfSides) {
this.numOfSides = numOfSides; 
coordinates = new ArrayList<>();	
}	
```

this is for you to implement this constructor of Triangle by just calling `super(3);`
but since you deleted that part of the starter code. It is perfectly fine to implement this constructor like this.